--- libpng-1.6.37/contrib/pngminus/Makefile~	2019-04-14 20:10:32.000000000 +0200
+++ libpng-1.6.37/contrib/pngminus/Makefile	2019-04-23 09:25:47.422459927 +0200
@@ -8,7 +8,7 @@
 RM = rm -f
 
 PNGINC = -I../..
-PNGLIB_SHARED = -L../.. -lpng
+PNGLIB_SHARED = -L../.. -L../../.libs/ -lpng16
 PNGLIB_STATIC = ../../libpng.a
 
 # Uncomment the following if you have a custom zlib build at ../../../zlib
@@ -20,10 +20,10 @@
 ZLIB_SHARED = -lz
 ZLIB_STATIC = -lz
 
-CPPFLAGS = $(PNGINC) $(ZINC)
+CPPFLAGS = $(OPT_FLAGS) $(PNGINC) $(ZINC)
 CFLAGS =
 LDFLAGS =
-LIBS_SHARED = $(PNGLIB_SHARED) $(ZLIB_SHARED)
+LIBS_SHARED = $(LD_FLAGS) $(PNGLIB_SHARED) $(ZLIB_SHARED)
 LIBS_STATIC = $(PNGLIB_STATIC) $(ZLIB_STATIC)
 
 EXEEXT =
@@ -31,7 +31,7 @@
 
 # dependencies
 
-all: png2pnm$(EXEEXT) pnm2png$(EXEEXT) png2pnm-static$(EXEEXT) pnm2png-static$(EXEEXT)
+all: png2pnm$(EXEEXT) pnm2png$(EXEEXT)
 
 png2pnm.o: png2pnm.c
 	$(CC) -c $(CPPFLAGS) $(CFLAGS) png2pnm.c
