--- libpng-1.6.20/tests/pngstest.orig	2015-09-26 00:48:24.000000000 +0200
+++ libpng-1.6.20/tests/pngstest	2015-12-05 18:52:37.106053733 +0100
@@ -17,7 +17,8 @@
 shift
 alpha="$1"
 shift
-exec ./pngstest --tmpfile "${gamma}-${alpha}-" --log ${1+"$@"} $(
+
+loop() {
    for f in "${srcdir}/contrib/testpngs/"*.png
    do
       g=
@@ -49,4 +50,6 @@
             test "$alpha" = "opaque" -o "$alpha" = "none" && echo "$g";;
       esac
    done
-)
+}
+
+exec ./pngstest --tmpfile "${gamma}-${alpha}-" --log ${1+"$@"} $(loop)
