--- libpng-1.5.7/contrib/pngminus/makefile.std.orig	2011-12-15 16:45:32.000000000 +0100
+++ libpng-1.5.7/contrib/pngminus/makefile.std	2011-12-18 16:24:38.099367091 +0100
@@ -12,20 +12,20 @@
 #PNGLIB = -L$(PNGPATH)/lib -lpng15
 #PNGLIBS = $(PNGPATH)/lib/libpng15.a
 PNGINC = -I../..
-PNGLIB = -L../.. -lpng
+PNGLIB = -L../.. -L../../.libs -lpng15
 PNGLIBS = ../../libpng.a
 
 #ZPATH = /usr/local
 #ZINC = -I$(ZPATH)/include
 #ZLIB = -L$(ZPATH)/lib -lz
 #ZLIBS = $(ZPATH)/lib/libz.a
-ZINC = -I../../../zlib
-ZLIB = -L../../../zlib -lz
-ZLIBS = ../../../zlib/libz.a
+ZINC =
+ZLIB = -lz
+ZLIBS = -lz
 
-CFLAGS=$(PNGINC) $(ZINC)
+CFLAGS=$(OPT_FLAGS) $(PNGINC) $(ZINC)
 LDLIBS=$(PNGLIB) $(ZLIB)
-LDLIBSS=$(PNGLIBS) $(ZLIBS)
+LDLIBSS=$(LD_FLAGS) $(PNGLIBS) $(ZLIBS)
 C=.c
 O=.o
 L=.a
@@ -33,8 +33,7 @@
 
 # dependencies
 
-#all: png2pnm$(E) pnm2png$(E)
-all: png2pnm$(E) pnm2png$(E) png2pnm-static$(E) pnm2png-static$(E)
+all: png2pnm$(E) pnm2png$(E)
 
 png2pnm$(O): png2pnm$(C)
 	$(CC) -c $(CFLAGS) png2pnm$(C)
