--- libpng-1.5.7/scripts/symbols.def.orig	2011-12-15 16:45:34.000000000 +0100
+++ libpng-1.5.7/scripts/symbols.def	2011-12-18 19:44:57.326436207 +0100
@@ -239,4 +239,24 @@
  png_get_cHRM_XYZ_fixed @231
  png_set_cHRM_XYZ @232
  png_set_cHRM_XYZ_fixed @233
  png_set_check_for_invalid_index @234
+ png_get_acTL @235
+ png_set_acTL @236
+ png_get_num_frames @237
+ png_get_num_plays @238
+ png_get_next_frame_fcTL @239
+ png_set_next_frame_fcTL @240
+ png_get_next_frame_width @241
+ png_get_next_frame_height @242
+ png_get_next_frame_x_offset @243
+ png_get_next_frame_y_offset @244
+ png_get_next_frame_delay_num @245
+ png_get_next_frame_delay_den @246
+ png_get_next_frame_dispose_op @247
+ png_get_next_frame_blend_op @248
+ png_get_first_frame_is_hidden @249
+ png_set_first_frame_is_hidden @250
+ png_read_frame_head @251
+ png_set_progressive_frame_fn @252
+ png_write_frame_head @253
+ png_write_frame_tail @254
